lib/sprite.dart

```diff
@@ -8,7 +8,7 @@ import 'palette.dart';
 class Sprite {
   Paint paint = BasicPalette.white.paint;
   Image image;
-  Rect src;
+  Rect src = Rect.zero;
```

Comment:
hum should the default be the whole image?

Reply:
It is. see `srcSize`/`srcPosition` getters and setters. This default allows those to work as intended while avoiding mind-bending null checks on src.